ovos_bus_client/client/async_client.py

```
@@ -0,0 +1,400 @@
+    async def collect(self) -> List[Message]:
+        """Emit the query and wait for all registered handlers to respond."""
+        self._setup()
+        await self.bus.emit(self.message)
+        await asyncio.sleep(self.min_timeout)
+
+        if not self.handlers:
+            self._teardown()
+            return []
+
+        # Wait for all handlers up to max_timeout
+        time_waited = self.min_timeout
```

Ruff (0.15.5)

[error] Local variable `time_waited` is assigned to but never used

Remove assignment to unused variable `time_waited`

(F841)